### PR TITLE
Fix dbotscore type file

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/24514

## Description
Fixed the DbotScore type file in this integrations(non 5.5 versions):
- Virustotal
- Virustotal-Private-api
- OPSWAT-Metadefender V2

## Screenshots
Screen Shot 2020-05-14 at 16 01 14

## notes
Couldn't check on OPSWAT-Metadefender V2 - no instance

### CODE REVIEW

**Comment by ronykoz** [comment]
fixes https://github.com/demisto/etc/issues/24514

**Comment by barchen1** [comment]
@ronykoz the other ones didn't use dt in context

**Comment by anara123** [comment]
The proper fix should be that we check if demisto version is less than 5.5.0 then use DBotScore and above use with DT
For that we have the following function in CommonServerPython `is_demisto_version_ge('5.5.0')`

File: Packs/VirusTotal-Private_API/Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py [hunk 1]
```diff
@@ -221,6 +221,12 @@ def create_file_output(file_hash, threshold, vt_response, short_format):
     return entry
 
 
+def get_dbot_file_context(file_hash, dbotscore):
+
+    return [{'Indicator': file_hash, 'Type': 'hash', 'Vendor': 'VirusTotal - Private API', 'Score': dbotscore},
```
Comment (anara123) [inline Packs/VirusTotal-Private_API/Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py:226]: The type should be `file` and not `hash`

File: Packs/opswat-metadefender/Integrations/integration-opswat-metadefender-v2.yml [hunk 2]
```diff
@@ -129,6 +129,12 @@ script:
         res = http_req(method='GET', url_suffix=f'file/{scan_id}')
         return res
 
+    def get_dbot_file_context(file_hash, dbotscore):
+
+        return [{'Indicator': file_hash, 'Type': 'hash', 'Vendor': 'OPSWAT', 'Score': dbotscore},
```
Comment (anara123) [inline Packs/opswat-metadefender/Integrations/integration-opswat-metadefender-v2.yml:134]: Same here, we don't need Type: 'hash', remove it .
Leave just the file

File: Packs/NonSupported/Integrations/integration-VirusTotal.yml [hunk 3]
```diff
@@ -245,8 +245,8 @@ script:
         var o = res.obj;
         var ec = {};
         if (o.response_code === 0) {
-            ec["DBotScore(val.Indicator && val.Indicator === obj.Indicator && val.Vendor === obj.Vendor && val.Type === obj.Type)"] = {Indicator: hash, Type: 'hash', Vendor: 'VirusTotal', Score: 0};
-            ec["DBotScore(val.Indicator && val.Indicator === obj.Indicator && val.Vendor === obj.Vendor && val.Type === obj.Type)"] = {Indicator: hash, Type: 'file', Vendor: 'VirusTotal', Score: 0};
+            ec["DBotScore"] = [{Indicator: hash, Type: 'hash', Vendor: 'VirusTotal', Score: 0},
```
Comment (anara123) [inline Packs/NonSupported/Integrations/integration-VirusTotal.yml:248]: remove `hash`

Comment (anara123) [reply Packs/NonSupported/Integrations/integration-VirusTotal.yml:248]: The fix in this file is ok. No need to remove hash.

Comment (barchen1) [reply Packs/NonSupported/Integrations/integration-VirusTotal.yml:248]: done

File: Packs/VirusTotal-Private_API/Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py (hunk 1 above)
Comment (barchen1) [reply Packs/VirusTotal-Private_API/Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py]: done

File: Packs/opswat-metadefender/Integrations/integration-opswat-metadefender-v2.yml (hunk 2 above)
Comment (barchen1) [reply Packs/opswat-metadefender/Integrations/integration-opswat-metadefender-v2.yml]: done

**Comment by ronykoz** [comment]
@barchen1 add a changelog for all your changes, you can write them with @roysagi 
Something like: `Fixed an issue with DbotScore output.`

**Pull request merged**